Makefile

```diff
@@ -52,14 +57,36 @@ deps.linux.amd64: duckdb
 .PHONY: deps.linux.arm64
 deps.linux.arm64: duckdb
 	if [ "$(shell uname -s | tr '[:upper:]' '[:lower:]')" != "linux" ]; then echo "Error: must run build on linux"; false; fi
+	mkdir -p deps/linux_arm64
 
 	cd duckdb && \
 	CC="aarch64-linux-gnu-gcc" CXX="aarch64-linux-gnu-g++" CFLAGS="-O3" CXXFLAGS="-O3" ${DUCKDB_COMMON_BUILD_FLAGS} make bundle-library -j 2
 	cp duckdb/build/release/libduckdb_bundle.a deps/linux_arm64/libduckdb.a
 
+.PHONY: deps.freebsd.amd64
+deps.windows.amd64: duckdb
+	if [ "$(shell uname -s | tr '[:upper:]' '[:lower:]')" != "mingw64_nt-10.0-20348" ]; then echo "Error: must run build on windows"; false; fi
+	mkdir -p deps/windows_amd64
+	
+	# this is just code copied from duckdb and fixed for windows. Would like to not change this, and use `make bundle-library` once its fixed.
```

Comment:
I agree. But from my side, this is fine for this initial PR. We can turn it into a `FIXME: use bundle-library, once fixed for Windows` note.